.github/workflows/sdist.yml

```diff
@@ -54,6 +54,12 @@ jobs:
         pip list
         python setup.py sdist --formats=gztar
 
+    - name: Upload sdist artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: ${{ github.sha }}-${{matrix.python-version}}-sdist.gz
```

Comment:
You can just make the name ``sdist`` for simplicity. On builds from master, part of the commit sha is included in the package version.

Python version is probably not that important to include either.

Reply:
Thanks for the review! I agree that both of these bits of info aren't _that_ important, but they are kinda nice and I don't really see the drawback of having them? Is there a benefit of having a consistent artifact name?

I was imagining the scenario of testing a PR that changes the sdist. In that case, I was imagining that you would want to look at the sdist before the PR, and the sdist after the PR, and compare them. So you would have two copies on your local machine, and I was thinking the SHA would be nice to be able to keep them separated.

Also, if they are named the same, then each of the three actions runs will overwrite each other, and we will only get the sdist from one of the python versions. And it will be undeterministic, which ever uploads last. I can't really foresee a problem with that, but again it just seems safer to keep all three, named precisely, without a big drawback.

I can change though if you confirm, just wanted to explain my reasoning. Thanks!

Reply:
I forgot that there are three sdists being built for each python version. Thanks for the reminder. We can keep the python version in the name, but I would still like to remove the hash. 

My main issue with the hash is that it makes the artifact name hard to read and is redundant, since the sdist inside the artifact has part of the commit hash(it's named something like ``pandas-1.5.0.dev0+500.gffa07fb96d.tar.gz``(partial hash is the part after "g")

Reply:
Thanks @lithomas1, sounds good! I just pushed the fix